vasyl_varyshchuk/Week_1/task_3/generate_file_size.py

```diff
@@ -0,0 +1,50 @@
+"""
+This module returns random size of file
+"""
+
+import random
+
+list_of_files = ['reports.txt', 'monthly_reports.zip', 'README.txt',
+                 'picture.jpg', 'logo.ttf', 'config.ini']
+
+
+def recognize_file_type(a_file):
+    return a_file.split('.')[1]
```

Comment:
Why don't you use something like this:

    filename, extension = a_file.split('.')

?

Reply:
This is a good solution, but if I do not use the filename variable in the code below, is it expedient 
 to assign it some value?

Reply:
You should use `_, extension = a_file.split('.')`, but I don't like this way.

Also, both variants are more Pythonic than yours.

P.S. `a_file` - is a bad name for variable.

Reply:
In most examples I have found, it is recommended to use _ for ignoring the specific values.

Reply:
The first googled approved solution:
https://stackoverflow.com/a/541394/7335432

The second googled solution:
https://www.systutorials.com/241539/how-to-get-the-file-extension-from-a-filename-in-python/

Reply:
In both cases, they are using unpacking for extension extraction.

Reply:
Thanks, it was useful for me. I used the similar approach in my code.